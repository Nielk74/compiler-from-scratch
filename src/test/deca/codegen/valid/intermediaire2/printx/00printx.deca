{
    float x = 0x1.FFFFFEp0;
    printx(x);
}