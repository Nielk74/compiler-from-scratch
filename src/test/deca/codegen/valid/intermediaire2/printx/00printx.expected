0x1.fffffep+00x1.fffffep+0
